scripts/restart_resque.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Stops and then starts resque-pool in either production or development environment
+# script/restart_resque.sh [production|development]
+
+RESQUE_POOL_PIDFILE="$(pwd)/tmp/pids/resque-pool.pid"
```

Comment:
Perhaps we should have a comment in the main comment block stating from where the script should be run?